src/util/win32_utils.cc

```diff
@@ -0,0 +1,17 @@
+#include "win32_utils.h"
+
+#define NOMINMAX
+#include <windows.h>
+
+namespace jumanpp {
+namespace util {
+
+std::wstring to_wide_string(const StringPiece str) {
+  int size = MultiByteToWideChar(CP_UTF8, 0, str.data(), -1, NULL, 0);
```

Comment:
OK, this is incorrect, there is no guarantee that StringPiece will be null-terminated. 
But it always knows its `size()` on the other hand.

Reply:
Oh, ok. Sorry i thought it is usually constructed from chars, but there is also slicing.